core/templates/components/common-layout-directives/navigation-bars/top-navigation-bar.component.ts

```diff
@@ -540,6 +540,21 @@ export class TopNavigationBarComponent implements OnInit, OnDestroy {
     );
   }
 
+  navigateToAboutPage(): void {
+    this.siteAnalyticsService.registerClickNavbarButtonEvent('About');
```

Comment:
Use the enum directly instead of using its string representation.
```suggestion
    this.siteAnalyticsService.registerClickNavbarButtonEvent(ABOUT);
```